chapter-05/start/MainGame.cs

```diff
@@ -45,6 +41,11 @@ public MainGame()
         /// </summary>
         protected override void Initialize()
         {
+            graphics.PreferredBackBufferWidth = 1024;
+            graphics.PreferredBackBufferHeight = 768;
+            graphics.IsFullScreen = false;
```

Comment:
Was set to fullscreen previously

Reply:
yes, it was. I was surprised by the fullscreen mode with no way to exit the game. Now that I think about it, though, maybe then enter key was what I needed to press? I ended up alt-tabbing back to visual studio to stop the process. I also went back and read the chapter and thought the end solution could have fullscreen = false after you explained to the reader that this was a possibility. 

I can revert it back to true if you want.

Reply:
Good eye. I should've mentioned this to you in my email...